README.md

```diff
@@ -1 +1,128 @@
+
+## Registering a project
+
+If you want to register your project with the `central-uploader`, you need to add a section
+as below to the `products.json` file, and run the pipelines.
+
+```
+    {
+        "name": "opensearch",
+        "lp-building-project": "soss",
+        "lp-releasing-project": "opensearch-releases",
+        "lp-building-repo": "~data-platform/soss/+source/opensearch-build/+git/opensearch-build/",
+        "lp-building-branch-prefix": "lp-2",
+        "lp-consumer-key": "LP_CONSUMER_KEY",
+        "lp-access-token": "LP_ACCESS_TOKEN",
+        "lp-access-secret": "LP_SECRET_TOKEN",
+        "artifactory-url": "https://canonical.jfrog.io/artifactory/dataplatform-opensearch-maven-local/",
+        "artifactory-user": "ARTIFACTORY_USER",
+        "artifactory-token": "ARTIFACTORY_TOKEN",
+        "tarball-regex": "opensearch-*.tar.gz"
+    }
+```
+
+Note that
+ - the `lp-building-branch-prefix` has to correspond to the release branch naming terminology of
+   of the Project's SOSS build repository.
+ - `tarball-regex` has to correspond to the artifactory output name
+ - the `lp-consumer-key`, `lp-access-token`, `lp-access-secret`, `artifactory-user`, `artifactory-token` fields
+   are to be defined as in the example above
+
+
+## Publishing a release
+
+There are regular executions of the `central-uploader` Release pipelines, but they can also be triggered manually.
+
+Within this pipeline 
+
+ 1. Existing artifacts are downloaded from the project's Launchpad SOSS build space
+ 2. A new Github Release is created for the project within the scope of the `central-uploader` repository
+ 3. A new tag is also created with the release here
+ 4. The artifacts are uploaded to the project's public Launchpad releases space
+
+## Testing
+
+When triggering your first pipeline runs, keep in mind that a release is only to be published once.
+If you would like to re-run the publishing method for testing purposes, you need to make sure that
+starting from item 2. of the list above all traces of the previous publish attemt under the same version are removed.
+
+# Project Built from Source (prerequisite)
+
+In order to be able to use `central-uploader`, you need to make sure that your project complies to the following criteria.
+
+
+### 0. Launchpad public git repo (optional)
+
+Your project's source code has to reside in a Launchpad git repository
+
+In case you may have multiple repositories to be bundled up in a single build, those repositories have to reside in
+the Launchpad public git space, available to anyone who has a Launchpad account
+
+In case your project repository is the same space where you want to apply your build mechanism, you can skip this step.
+
+Example: https://code.launchpad.net/~medib/opensearch-project-components/+git/opensearch-performance-analyzer
+
+
+### 1. Launchpad build repository
+
+The repository where the build process is to be performed has to reside in the Launchpad SOSS space.
```

Comment:
This is also quite use-case specific. I mean: we need to put the repository in this space because we want to use a canonical Jfrog, and the credentials for accessing to this are only injected by the LP team into the projects of the SOSS space. 

But again, this is not really relevant to the central uploader.